docs/src/fs_event.rst

```diff
@@ -55,7 +54,6 @@ Data types
             * or kqueue to detect events. This may not work on remote filesystems such
             * as NFS mounts. This flag makes fs_event fall back to calling stat() on a
             * regular interval.
-            * This flag is currently not implemented yet on any backend.
```

Comment:
If my experiences with the node.js / io.js documentation have taught me anything, it's that a) you can never be too explicit, b) there is no such thing as redundancy, and c) you can never be too explicit.  I would suggest leaving them in but LGTM either way.